vscode/src/webview/webview.tsx

```diff
@@ -15,12 +15,13 @@ import {
   type ReData,
 } from "qsharp-lang/ux";
 import { HelpPage } from "./help";
+import { DocumentationView } from "./docview";
 
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-ignore - there are no types for this
 import mk from "@vscode/markdown-it-katex";
 import markdownIt from "markdown-it";
-const md = markdownIt("commonmark", { html: true, breaks: true });
+const md = markdownIt("commonmark");
```

Comment:
The options were the default and redundant, so I removed them for consistency with other usage in the code.